tests/unit/test_viewmodel.py

```diff
@@ -0,0 +1,45 @@
+from viewmodel import ViewModel
+import pytest
+from datetime import datetime
+import dateutil.parser
+from card import Card
+
+vm = ViewModel(["my todo items"], ["my doing items"], ["my done items"])
+last_modified = '2020-09-20 11:18:26.051811'
+today_mock = '2020-10-01 00:16:26.061713'
+future_date = '2020-10-25 00:16:26.061713'
```

Comment:
Do you expect your tests to still pass? Because they don't.

The result of a test shouldn't depend on what day you run it. Either create these dates based on the actual current date (not perfect because some tests could still fail if the depend on day of the week or daylight savings but this is often good enough), or ideally install and use something like freezegun to mock the current datetime https://pypi.org/project/freezegun/

Reply:
Done